modules/programs/zsh.nix

```
@@ -152,6 +152,17 @@ let
           Name of the theme to be used by oh-my-zsh.
         '';
       };
+
+      extraSettings = mkOption {
```

The standard name for textual extra configuration is `extraConfig` so can here change
```suggestion
      extraConfig = mkOption {
```